medicines/web/src/components/search-results/index.tsx

```diff
@@ -113,18 +118,36 @@ const StyledDrugList = styled.div`
     font-weight: bold;
     font-style: normal;
   }
-`;
 
-export interface IDocument {
-  activeSubstances: string[];
-  context: string;
-  created: string;
-  docType: string;
-  fileSize: string;
-  name: string;
-  product: string;
-  url: string;
-}
+  .column {
+    float: left;
+  }
+  .filter {
+    width: 40%;
+  }
```

Comment:
May be worth setting a max width on this - 40% on a wide screen is gonna look ments. Similarly but less importantly, a min width?

Reply:
Considered using something like flexbox to control widths more fluidly across different size screens and devices?

Reply:
Using flex now. Switches direction to `column` in mobile view, and a width is no longer specified so the amount of real estate on a large screen isn't bizarre now.